src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
```

Comment:
Log of all environment variables being available as part of release logs was very useful to debug issues. Wondering if this facility is available within Build also. If so, we should reuse the functionality between Build & Release so that customer experience is similar? If not, should we have this functionality in Build also (in which case, moving it to Agent.Common.Util makes sense)?